src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -372,7 +374,8 @@ public Object onTraversalSuccess() {
 
     private void generateIntervalsVCFFileFromAllShards() {
         logger.info("Generating intervals VCF file...");
-        final VariantContextWriter intervalsVCFWriter = createVCFWriter(outputIntervalsVCFFile);
+        final VariantContextWriter intervalsVCFWriter = GATKVariantContextUtils.createVCFWriter(
```

Comment:
The GATKTool method won't accept a sequence dictionary.  It relies on the engine discovering a sequence dictionary from one of the inputs, which doesn't work so well here because the inputs are specified as directories.

Reply:
Isn’t the dictionary hooked up now via getBestAvailableSequenceDictionary?

Reply:
Ah yes, this is a holdover from before I implemented that suggestion Louis had.  Thanks for the extra set of eyes.